### PR TITLE
Fix issue82 and issue86

### PR DESCRIPTION
**Pull request by bmesnet**
Fix issue #86
It was finally just a typo in a test of the hw part
Adding by the way hls_memcopy_512 as a new example and cleaning the unnecessary comments

### CODE REVIEW

**Comment by bmesnet** [comment]
Fix issue #82

File: actions/hls_memcopy_1024/README.md [hunk 1]
```diff
@@ -7,5 +7,5 @@
   * Block RAM memory (memory inside the FPGA itself)
 * The example code details the copy mechanism.
 
-:star: Please check the [actions/hls_memcopy/doc](./doc/) directory for detailed information
+:star: Please check the [actions/hls_memcopy/_1024doc](./doc/) directory for detailed information
```
Comment (acastellane) [inline actions/hls_memcopy_1024/README.md:11]: This documentation has been moved to https://opencapi.github.io/oc-accel-doc/actions-doc/hls_memcopy_1024/

Comment (bmesnet) [reply actions/hls_memcopy_1024/README.md:11]: corrected for memcopy 1024 and 512...
Will need to create 512 doc and correct 1024 doc with right numbers
Thx

**Review by acastellane — DISMISSED** [review]

**Review by acastellane — APPROVED** [review]
Single_engine happens to fail from time to time in the middle of the test. This seems not related to the current proposal.

**Pull request merged**